engine/table/src/test/java/io/deephaven/engine/table/impl/TestAddOnlyToBlinkTableAdapter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+
+package io.deephaven.engine.table.impl;
+
+import io.deephaven.engine.context.ExecutionContext;
+import io.deephaven.engine.rowset.RowSetFactory;
+import io.deephaven.engine.rowset.WritableRowSet;
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.testutil.junit4.EngineCleanup;
+import io.deephaven.util.SafeCloseable;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static io.deephaven.engine.testutil.TstUtils.*;
+import static io.deephaven.engine.util.TableTools.intCol;
+import static io.deephaven.engine.util.TableTools.stringCol;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestAddOnlyToBlinkTableAdapter {
+
+    public final EngineCleanup base = new EngineCleanup();
+    private CapturingUpdateGraph updateGraph;
+    private SafeCloseable contextCloseable;
+
+    @BeforeEach
+    public void setup() throws Exception {
+        base.setUp();
+        updateGraph = new CapturingUpdateGraph(ExecutionContext.getContext().getUpdateGraph().cast());
+        contextCloseable = updateGraph.context.open();
+    }
+
+    @AfterEach
+    public void tearDown() throws Exception {
+        contextCloseable.close();
+        base.tearDown();
+    }
```

Comment:
This version works for me:
```suggestion
import org.junit.After;
import org.junit.Before;
import org.junit.Rule;
import org.junit.Test;

import static io.deephaven.engine.testutil.TstUtils.*;
import static io.deephaven.engine.util.TableTools.intCol;
import static io.deephaven.engine.util.TableTools.stringCol;
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertTrue;

public class TestAddOnlyToBlinkTableAdapter {

    @Rule
    public final EngineCleanup base = new EngineCleanup();

    private CapturingUpdateGraph updateGraph;
    private SafeCloseable contextCloseable;

    @Before
    public void setup() throws Exception {
        updateGraph = new CapturingUpdateGraph(ExecutionContext.getContext().getUpdateGraph().cast());
        contextCloseable = updateGraph.context.open();
    }

    @After
    public void tearDown() throws Exception {
        contextCloseable.close();
    }
```